src/views/questions/View.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div v-if="loading">Loading</div>
+
+    <div v-else class="container flex flex-wrap p-8 mx-auto">
+      <question :question="question" />
+
+      <div class="text-2xl">
+        {{ question.answer_count }} Answers
+        <answer v-for="answer in answers" :key="answer.id" :answer="answer" />
+
+        <comment v-for="comment in comments" :key="comment.id" :comment="comment" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { getQuestions } from "@/api/questions";
+import { getAnswers } from "@/api/answers";
+import { getComments } from "@/api/comments";
+import Question from "@/components/answered-question/Question.vue";
+import Answer from "@/components/answered-question/Answer.vue";
+import Comment from "@/components/answered-question/Comment.vue";
+
+export default {
+  components: {
+    Question,
+    Answer,
+    Comment,
+  },
+  props: {
+    id: {
+      type: [String, Number],
+      required: true,
+    },
+  },
+  data: () => ({
+    question: null,
+    loading: true,
+    answers: [],
+    comments: [],
+  }),
+  async created() {
+    await this.fetchQuestion();
+    await this.fetchAnswers();
+    await this.fetchComments();
```

Comment:
Here you have 3 queries: question, answers and comments. But each next request will start when previous will be finished. Better to start all 3 together in parallel and change `loading` when all 3 will finish. See on `Promise.all()` instead of async/await here.

Reply:
Fixed